### PR TITLE
Small DB refactor

### PR DESCRIPTION
**Pull request by frozeman**
This PR fixes the tests from the "tests" folder.
Refactores the DB to be called "syncDB", rather than "syncMinimongo".

i moved a few places around and improved some IPC channel names

### CODE REVIEW

File: modules/windows.js [hunk 1]
```diff
@@ -260,6 +258,15 @@ class Windows {
...
 
+        // always show on top of main window
+        let parent = _.find(this._windows, (w) => {
+            return w.type === 'main';
+        });
+
+        if(parent)
+            opts.electronOptions.parent = parent.window;
+
+
```
Comment (evertonfraga) [inline modules/windows.js]: question: wasn't it the default behavior?

File: modules/preloader/popupWindows.js [hunk 2]
```diff
@@ -17,8 +18,6 @@ const web3Admin = require('../web3Admin.js');
 
 require('./include/setBasePath')('interface');
 
-// register with window manager
-ipc.send('backendAction_setWindowId');
```
Comment (hiddentao) [inline modules/preloader/popupWindows.js]: Why are these calls being removed? These are essential for hooking up the window id (and does event sending) correctly. Also, noticeably you didn't remove the event handler for this event from the Windows manager.

Comment (frozeman) [inline modules/preloader/popupWindows.js]: i moved them into commons.
As we fire them in all preloaders, anyway.

Comment (hiddentao) [inline modules/preloader/popupWindows.js]: Apologies, didn't spot that line.

**Comment by frozeman** [comment]
I will move the wallet "test" to "tests". So don't merge just yet

**Comment by frozeman** [comment]
I moved the tests, but they don't run anymore. The wallet seems to want to start and then quits.
This kills spectator:
```
 1) basic.test.js "before all" hook: before:
     waitUntilWindowLoaded Cannot read property 'isLoading' of undefined
  TypeError: Cannot read property 'isLoading' of undefined
      at Object.<anonymous> (node_modules/spectron/lib/application.js:216:30)
      at Timer.tick (node_modules/webdriverio/build/lib/utils/Timer.js:111:31)
      at Timer.start (node_modules/webdriverio/build/lib/utils/Timer.js:75:22)
      at new Timer (node_modules/webdriverio/build/lib/utils/Timer.js:62:14)
      at Object.exports.default (node_modules/webdriverio/build/lib/commands/waitUntil.js:34:17)
      at Object.exec (node_modules/webdriverio/build/lib/helpers/safeExecute.js:28:24)
      at Object.resolve (node_modules/webdriverio/build/lib/webdriverio.js:189:29)
      at lastPromise.then.resolve.call.depth (node_modules/webdriverio/build/lib/webdriverio.js:433:32)
      at _fulfilled (node_modules/q/q.js:834:54)
      at self.promiseDispatch.done (node_modules/q/q.js:863:30)
      at Promise.promise.promiseDispatch (node_modules/q/q.js:796:13)
      at node_modules/q/q.js:556:49
      at runSingle (node_modules/q/q.js:137:13)
      at flush (node_modules/q/q.js:125:13)
      at _combinedTickCallback (internal/process/next_tick.js:67:7)
      at process._tickCallback (internal/process/next_tick.js:98:9)
```
@hiddentao any idea?

File: test/.gitignore [hunk 3]
```diff
@@ -1,2 +0,0 @@
-mist.log
```
Comment (evertonfraga) [inline test/.gitignore]: this mist.log

**Comment by evertonfraga** [comment]
@frozeman ready to be merged.

**Pull request merged**